src/renderer/data/SelectedAccountsList/selectedaccountsSlice.ts

```diff
@@ -0,0 +1,118 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+// https://redux.js.org/usage/usage-with-typescript#define-slice-state-and-action-types
+// eslint-disable-next-line import/no-cycle
+import { RootState } from '../../store';
+import { loadState } from '../localstorage';
+import { AccountInfo } from '../accounts/accountsSlice';
+
+export interface SelectedAccountsList {
+  pinnedAccounts: string[]; // list of pubKeys (TODO: should really add net...)
+  selectedAccount: string;
+  hoveredAccount: string;
+  editedAccount: string;
+  rootKey: string;
+}
+
+// Define the initial state using that type
+let initialState: SelectedAccountsList = {
+  pinnedAccounts: [],
+  selectedAccount: '',
+  hoveredAccount: '',
+  editedAccount: '',
+  rootKey: '',
+};
+const loaded = loadState('selectedaccounts');
```

Comment:
`selectedAccounts`, etc, couple other instances of that in this file if you could please address

Reply:
slice names seem to be done as all lowercase - so it seems more consistent to me to do the same thing for saving the slice into the localState

Reply:
mmm, nope, can't find them, we'll need to talk about this so i have a chance of groking.